pywbem/_statistics.py

```diff
@@ -141,25 +221,26 @@ def container(self):
     @property
     def count(self):
         """
-        :term:`integer`: The number of measured operations (that is,
-        invocations of the :meth:`~pywbem.OperationStatistic.stop_timer`
-        method).
+        :term:`integer`: The number of measured operations.
         """
         return self._count
 
     @property
     def exception_count(self):
         """
-        :term:`integer`: The number of exceptions that occurred when invoking
-        the measured operations.
+        :term:`integer`: The number of measured operations that resulted in an
+        exception returned to the pywbem user (for example because a failure
+        was indicated in the operation response of the WBEM server, or because
+        pywbem itself detected a failure before sending the request or after
+        receiving the response).
         """
         return self._exception_count
 
     @property
     def avg_time(self):
         """
-        :class:`py:float`: The average elapsed time for invoking the measured
-        operations, in seconds.
+        :class:`py:float`: The average elapsed client time for invoking the
```

Comment:
I think it sounds better if we change the phrase "for invoking" to " for execution of" so it becomse "for execution of the measured operations"  This applies to the min_time and max time also

Reply:
DONE